openhands/server/session/agent.py

```diff
@@ -41,11 +47,18 @@ async def start(
         agent_to_llm_config: dict[str, LLMConfig] | None = None,
         agent_configs: dict[str, AgentConfig] | None = None,
     ):
-        """Starts the agent session.
+        """Starts the Agent session
 
-        Args:
-            start_event: The start event data (optional).
+        Parameters:
+        - runtime_name (str): The name of the runtime associated with the session
+        - config (AppConfig):
+        - agent (Agent):
+        - max_interations (int):
+        - max_budget_per_task (float):
+        - agent_to_llm_config (dict[str, LLMConfig]):
+        - agent_configs (dict[str, AgentConfig]):
```

Comment:
FWIW we're not consistent, but we have kind of settled or tried to not include the args types in docstrings. They're visible in hints, and prone to become outdated in docstrings.

Reply:
Removed types.